packages/mobile/src/app/App.tsx

```diff
@@ -58,9 +60,33 @@ const Modals = () => {
 }
 
 const App = () => {
+  const [
+    isPendingMandatoryCodePushUpdate,
+    setIsPendingMandatoryCodePushUpdate
+  ] = useState(false)
+
+  const codePushSync = useCallback(() => {
+    codePush.sync(
```

Comment:
Just checking it's cool if this is called multiple times throughout app lifecycle?

Reply:
yes def cool! though the way i wrote this, i assumed it would only run on app start and then again every time you foreground the app. is that not what's going to happen here?

Reply:
yeah that's what's happening :)

Reply:
oh suite